content/en/docs/setup/upgrade/helm/index.md

```diff
@@ -96,6 +96,32 @@ gateways is [actively in development](/docs/setup/upgrade/gateways/) and is cons
     $ helm upgrade istio-base manifests/charts/base -n istio-system --skip-crds
     {{< /text >}}
 
+### Revision Tags (experimental)
```

Comment:
Let's use consistent heading across this and `istioctl` based install guide.

`Stable revision labels (experimental)`

Reply:
fixed